packages/squiggle-lang/src/PointSet/PointSet.ts

```diff
@@ -48,13 +48,13 @@ export interface PointSet<T> {
   downsample(length: number): T;
   truncate(left: number | undefined, right: number | undefined): T;
 
-  updateIntegralCache(cache: ContinuousShape | undefined): T;
-
   integral(): ContinuousShape;
-  integralEndY(): number;
+  integralSum(): number;
   integralXtoY(x: number): number;
   integralYtoX(y: number): number;
 
+  withIntegralSum(sum: number): T; // force a given sum (useful for normalization)
```

Comment:
how about `adjustToTargetIntegralSum` or something else that makes it clearer that this forcing is hapening?

<img width="868" alt="image" src="https://user-images.githubusercontent.com/377065/227001985-78cd0b88-756d-4376-b844-f2e6c472b99d.png">

Reply:
`forceIntegralSum` is the version I wanted to go with initially, but it makes it less clear that this function returns a new object and doesn't modify the original.

`withForcedIntegralSum` or `withAdjustedIntegralSum` might be optimal.

Reply:
I like ``withForcedIntegralSum``

Reply:
Renamed, but I don't like to use named args when there's just one arg with obvious meaning and there probably won't be more.